configure

```diff
@@ -114,7 +128,12 @@ done
 while true; do
   # Configure the Cudnn version to use.
   if [ -z "$TF_CUDNN_VERSION" ]; then
-    read -p "Please specify the Cudnn version you want to use. [Leave empty to use system default]: " TF_CUDNN_VERSION
+    default_cudnn_version='6.5'
```

Comment:
Good catch - I had renamed the lib during development. The naming of the lib is a bit inconsistent between platforms. I'll make the default to be a platform dependent: 4.0.7 on Linux and 4 on OS X.